### PR TITLE
Makes svg2img an optional dependency

### PR DESCRIPTION
**Pull request by andreban**
- Prevents installation from failing when impossible to install
  svg2img.
- Code issues an error when generating a project that uses SVGs
  and the library is not installed.

### CODE REVIEW

**Comment by EtienneLem** [comment]
I believe this would fix using bubblewrap on Apple M1 devices, is that possible? Right now bubblewrap can’t be installed because [node-canvas](https://github.com/Automattic/node-canvas) fails to install, which I believe is a dependency of `svg2img`.

Not that it’s been rejected or anything, but any chance this PR could be reconsidered?

**Comment by andreban** [comment]
Hey @EtienneLem, thanks for bringing this up. I was not aware `node-canvas` doesn't work on Apple M1 devices.

My preference would be for the SVG code to not have any native dependencies, as I found those to frequently cause pain for developers using Bubblewrap - maybe by transforming SVGs into Android Drawables directly, instead of generating PNGs.

In the meantime, yes, I agree we should reconsider this PR. I don't actually recall why we didn't move forward with this. @PEConn do you recall?

**Review by PEConn — APPROVED** [review]
LGTM, although a bit more context in the error message about why svg2img isn't installed and what the user could do to fix it would be helpful.

File: packages/core/src/lib/wrappers/svg2img.ts [hunk 1]
```diff
@@ -34,9 +40,18 @@ export interface Svg2imgOptions {
...
 }
 
+export function isSvgSupported(): boolean {
+  return _svg2img !== undefined;
+}
+
 export function svg2img(svg: string, options: Svg2imgOptions = {}): Promise<Buffer> {
+  if (!_svg2img) {
+    return Promise.reject(
+        new Error('Failed to parse the SVG. The svg2img library is not installed'));
```
[the remark this answers is not in the document]
Comment (andreban) [reply packages/core/src/lib/wrappers/svg2img.ts]: Update. How does this look like?

**Review by PEConn — APPROVED** [review]
I still think the user will be left wondering "but *why* couldn't `svg2img` be installed", although I guess explaining that is more the responsibility of the `svg2img` team than us.

LGTM

**Pull request merged**